playroom/snippets.tsx

```diff
@@ -1069,7 +1071,7 @@ const exampleScreens: Array<Snippet> = [
       `,
     },
     {
-        group: '💎 Example Screens',
+        group: 'Ξ ✨ Example Screens',
```

Comment:
This reorganization change is caused by slow performance in big playroom examples when you try to open snippet menu.